pkg/converters/configmap/tcpservices.go

```diff
@@ -44,10 +44,11 @@ func NewTCPServicesConverter(options *convtypes.ConverterOptions, haproxy haprox
 }
 
 type tcpSvcConverter struct {
-	logger  types.Logger
-	cache   convtypes.Cache
-	haproxy haproxy.Config
-	changed *convtypes.ChangedObjects
+	logger               types.Logger
+	cache                convtypes.Cache
+	haproxy              haproxy.Config
+	changed              *convtypes.ChangedObjects
+	useEndpointSlicesAPI bool
```

Comment:
Missing initialization in the New...() constructor above.

Reply:
done